src/Components/benchmarkapps/Wasm.Performance/TestApp/Wasm.Performance.TestApp.csproj

```diff
@@ -8,7 +8,6 @@
       Clien caching isn't part of our performance measurement, so we'll skip it.
     -->
     <BlazorCacheBootResources>false</BlazorCacheBootResources>
-    <UseRazorSourceGenerator>false</UseRazorSourceGenerator>
```

Comment:
We don't need this anymore. I fixed the root cause in https://github.com/dotnet/aspnetcore/pull/33456 anyways.